### PR TITLE
Followup 1745: Edits to dev_guide/builds

### PR DESCRIPTION
**Pull request by bfallonf**
Followup to #1745 

@adellape @tnguyen-rh @tpoitras @ahardin-rh Let me know if there's issues.

### CODE REVIEW

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -906,47 +905,60 @@ source:
...
+start-build` with one of the `--from` options to provide the requisite binary
+data.
+
+The `*dockerfile*` and `*contextDir*` link:#source-code[source options] have
+special meaning with binary builds.
+
+`*dockerfile*` can be used with any binary build source. If `*dockerfile*` is
+used and the binary stream is an archive, its contents serve as a replacement
+Dockerfile to any Dockerfile in the archive. If `*dockerfile*` is used with the
+`--from-file` argument, and the file argument is named "Dockerfile", the value
```
Comment (tnguyen-rh) [inline dev_guide/builds.adoc]: I'd mark up "Dockerfile" as a file name, and remove the double-quotes.

File: dev_guide/builds.adoc [hunk 2]
```diff
@@ -906,47 +905,60 @@ source:
...
-the builder will store the data in a file with the same name at the top of the build context.
+* `--from-file`: The contents of the file you specify are sent as a binary
+stream to the builder. Then the builder stores the data in a file with the
+same name at the top of the build context.
 
-2) `--from-dir` and `--from-repo`:  The contents will be archived and sent as a binary stream to the builder.
-Then the builder will extract the contents of the archive within the build context directory.
+* `--from-dir` and `--from-repo`: The contents are archived and sent as a binary
+stream to the builder. Then the builder extracts the contents of the archive
+within the build context directory.
```
Comment (tnguyen-rh) [inline dev_guide/builds.adoc]: Above, the two instances of "Then the builder ..." sound weird to my ear.  Suggest: "The builder then ..."

**Comment by tnguyen-rh** [comment]
@bfallonf Couple of small suggestions, otherwise LGTM.

**Comment by bfallonf** [comment]
[rev_history]
|link:../dev_guide/builds.html[Builds]
|Added information about binary builds to the link:../dev_guide/builds.html#binary-source[Binary Source] section.
%

**Comment by bfallonf** [comment]
Thanks @tnguyen-rh . Merged.

**Pull request merged**